src/TestUtils/src/Microsoft.Maui.IntegrationTests/TemplateTests.cs

```diff
@@ -82,12 +82,12 @@ public void BuildMauiBlazorWebSolution(string framework, string config, string a
 		{
 			const string templateShortName = "maui-blazor-web";
 
-			var solutionProjectDir = TestDirectory;
+			var solutionProjectDir = TestDirectory + "+More";
```

Comment:
Unfortunately I can't do a proper end-to-end test of this here due to the bugs I linked in the description. But this change is better than nothing.